CHANGELOG.md

```diff
@@ -1,10 +1,32 @@
+2.0.0 (In Progress) - Release
+===================
+
+Breaking API Changes
+--------------------
+
+### Active call no longer accessible through the Device object.
+
+To future-proof the SDK, `Device.activeCall` is no longer available. Instead,
```

Comment:
Do we really need to mention "To future-proof the SDK"? I think we can remove this, it doesn't tell me why this is future proofing as the customer.